.jenkins/build.sh

```diff
@@ -51,9 +51,26 @@ if [[ "${BUILD_ENVIRONMENT}" == *-android* ]]; then
   exit 0
 fi
 if [[ "${BUILD_ENVIRONMENT}" == conda* ]]; then
+
+  # click (required by onnx) wants these set
+  export LANG=C.UTF-8
+  export LC_ALL=C.UTF-8
```

Comment:
This was needed to fix an exception thrown by 'click' (used by onnx) that complained about detection incorrect locale configuration on the current machine. These same lines appear https://github.com/caffe2/caffe2/blob/master/docker/jenkins/common/install_anaconda.sh#L6 but I guess the values don't persist.